src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -62,25 +62,47 @@ const ScheduleActivityPage = ({isLoggedIn, accessToken, userId, activity, links,
     }
 
     return (
-    <Grid container spacing={3} className={classes.root}>
-      <Grid item xs>
-        <Grid container justify="center" >
-          {randomArray.map((element) => <Button key={element.key} onClick={() => { alertUpdateActivity(element) }} size ="large" color="secondary" variant="outlined" className={classes.radio}> {element.title} </Button>)}
+    <div className={classes.root}>
+      <Grid container spacing={3}>
+        {randomArray.map((element) => 
+          <Grid item xs>
+            <Card className={classes.root}>
+
```

Comment:
One way to solve the duplicate issue is potentially adding <CardActionArea href={element.url}> </CardActionArea> so if the user clicks the top part of the card, they will get to the link instead of adding a button... maybe? or Maybe removing the div might help as well.

Reply:
Let me try that!